# HG changeset patch
# User Julien Bect <jbect@users.sourceforge.net>
# Date 1509520873 -3600
#      Wed Nov 01 08:21:13 2017 +0100
# Node ID 4e93c39b4b97e81dc440bb4f4cc2c64dc4b876de
# Parent  ec837ef7ee3b63156b36b60440de2529b92d2325
libinterp/parse-tree/module.mk: Add LLVM flags to libparse_tree_la

* libinterp/parse-tree/module.mk: Add LLVM flags to libparse_tree_la

diff --git a/libinterp/parse-tree/module.mk b/libinterp/parse-tree/module.mk
--- a/libinterp/parse-tree/module.mk
+++ b/libinterp/parse-tree/module.mk
@@ -106,17 +106,24 @@ PARSE_TREE_SRC = \
 %reldir%/oct-parse.yy: %reldir%/oct-parse.in.yy
 	$(AM_V_GEN)$(call subst-bison-api-decls,octave_)
 
 noinst_LTLIBRARIES += \
   %reldir%/libparse-tree.la
 
 %canon_reldir%_libparse_tree_la_SOURCES = $(PARSE_TREE_SRC)
 
-%canon_reldir%_libparse_tree_la_CPPFLAGS = $(libinterp_liboctinterp_la_CPPFLAGS)
+%canon_reldir%_libparse_tree_la_CPPFLAGS = \
+  $(libinterp_liboctinterp_la_CPPFLAGS) \
+  $(LLVM_CPPFLAGS)
 
-%canon_reldir%_libparse_tree_la_CFLAGS = $(AM_CFLAGS) $(WARN_CFLAGS)
+%canon_reldir%_libparse_tree_la_CFLAGS = \
+  $(AM_CFLAGS) \
+  $(WARN_CFLAGS)
 
-%canon_reldir%_libparse_tree_la_CXXFLAGS = $(AM_CXXFLAGS) $(WARN_CXXFLAGS)
+%canon_reldir%_libparse_tree_la_CXXFLAGS = \
+  $(AM_CXXFLAGS) \
+  $(WARN_CXXFLAGS) \
+  $(LLVM_CXXFLAGS)
 
 libinterp_EXTRA_DIST += \
   %reldir%/oct-parse.in.yy \
   %reldir%/octave.gperf
